### ISSUE TITLE
Hactoberfest'22

### ISSUE THREAD

**Issue by reveurguy** [GDSC-SGGSCC/Hacktoberfest-22#8]

**Comment by giftade** [comment]
I'd like to take this on

**Comment by reveurguy** [comment]
This is open for everyone. You can follow the instructions to contribute!

**Comment by Christian-Oleson** [comment]
I tackled Kotlin in https://github.com/GDSC-SGGSCC/Hacktoberfest-22/pull/22 along with a few minor Readme issues. Are there plans to add documentation to get folks started with each one of these languages? For example, on the JVM, you need to have a language installed and a valid JDK to run these hello world classes.

**Comment by shouryakhanka** [comment]
Can i add a program other then hello world?

**Comment by reveurguy** [comment]
> I tackled Kotlin in #22 along with a few minor Readme issues. Are there plans to add documentation to get folks started with each one of these languages? For example, on the JVM, you need to have a language installed and a valid JDK to run these hello world classes.

yes this can be implemented to this project as we go on in the future. We can add instructions for setting up the languages on our systems. That can make this repo a single-stop point to start learning programming languages!

**Comment by reveurguy** [comment]
> Can i add a program other then hello world?

currently we are focusing on hello world only. At a later stage we can look to add other basic programs.

**Comment by shouryakhanka** [comment]
But the hello world program has been printed in many languages, how can i contribute to this project?

**Comment by debxrshi** [comment]
Hi, you merged my pull request but its not showing up in my Hactoberfest profile? Does PR's to this repo count??

**Comment by reveurguy** [comment]
> But the hello world program has been printed in many languages, how can i contribute to this project?

You can search for new languages and submit a program in that. There would be many languages which we haven't covered already. That way you'll get to learn something new!

**Comment by reveurguy** [comment]
> Hi, you merged my pull request but its not showing up in my Hactoberfest profile? Does PR's to this repo count??

Yes the repository counts for Hacktoberfest, please check whether the account you have registered with is same as the one you made the PR on github and also check your profile on hacktoberfest website whether you have checked the participating as contributor option.
Also it can take some time for the PR to show up there so try to refresh the site.

**Comment by debxrshi** [comment]
Yes @reveurguy , it showed up. Apologies, this is my first time contributing to Hacktoberfest and so very new to this and learning! Thanks for accepting and merging my PR :)

**Comment by abhilipsasahoo03** [comment]
@reveurguy Hi can we include esoteric programming languages like FreakC or Bhai Lang?

**Comment by reveurguy** [comment]
> @reveurguy Hi can we include esoteric programming languages like FreakC or Bhai Lang?

No

**Comment by neeraj542** [comment]
can I work on this issue

**Comment by neeraj542** [comment]
Hii, I should work on this issue for my first Pull Request

**Comment by reveurguy** [comment]
> Hii, I should work on this issue for my first Pull Request

Hii, anyone is welcome to work on this. You can follow the guide to contribute!

**Comment by Raja-s-k** [comment]
Happy to inform that I am going to make my first pull request.

**Comment by rachana2522** [comment]
Can you please assign to me

**Comment by reveurguy** [comment]
> Hii, anyone is welcome to work on this. You can follow the guide to contribute!

Hii, anyone is welcome to work on this. You can follow the guide to contribute!

**Comment by rishabhyadav23** [comment]
Can i work on this issue?

**Comment by Mohitt0** [comment]
hey @reveurguy assign this to me
